### PR TITLE
Fix corrupt `iso extract minimal-iso` output to stdout; add tests

### PR DESCRIPTION
**Pull request by bgilbert**

### CODE REVIEW

File: src/live/mod.rs [hunk 1]
```diff
@@ -634,13 +634,13 @@ pub fn iso_extract_minimal_iso(config: IsoExtractMinimalIsoConfig) -> Result<()>
         .context("creating temporary file")?;
     data.unxzpack(full_iso.as_file()?, &mut outf)
         .context("unpacking miniso")?;
-    outf.seek(SeekFrom::Start(0))
-        .context("seeking back to start of miniso tempfile")?;
 
     modify_miniso_kargs(outf.as_file_mut(), config.rootfs_url.as_ref())
         .context("modifying miniso kernel args")?;
 
     if &config.output == "-" {
```
Comment (prestist) [inline src/live/mod.rs:641]: What does the presence of a "-" or absence of one mean ?

Comment (bgilbert) [reply src/live/mod.rs:641]: Specifying `-` as an input or output file is a common Unix command-line convention for stdin or stdout.

Comment (bgilbert) [reply src/live/mod.rs:641]: (But it's not magic; the program has to implement it explicitly.)

File: docs/release-notes.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ Major changes:
 Minor changes:
 
 - Detect truncated xz archives
+- iso extract minimal-iso: Fix corrupt output when writing to stdout
```
Comment (jlebon) [inline docs/release-notes.md:16]: Minor: WDYT about

```suggestion
- iso: Fix corrupt `extract minimal-iso` output when writing to stdout
```

for consistency?

(Came up while reviewing this PR with @prestist.)

**Review by jlebon — APPROVED** [review]
LGTM! Thanks a lot for adding tests.

**Review by prestist — APPROVED** [review]
Could not find anything, poked @jlebon to look at it with him. Def a LGTM! good job!

**Comment by bgilbert** [comment]
Rebased, updated changelog, and added tests for running `iso extract minimal-iso` on a customized ISO.

**Review by jlebon — APPROVED** [review]

**Pull request merged**